packages/gallery/src/components/item/buttons/customButton.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import CustomButtonIcon from './customButtonIcon.js';
-import utils from '../../../utils/index.js';
 import EVENTS from '../../../utils/constants/events';
 import { GalleryComponent } from '../../galleryComponent';
 
 export default class CustomButton extends GalleryComponent {
   render() {
     const { styleParams } = this.props;
-    const defaultText = utils.isStoreGallery() ? 'Buy Now' : 'Click here';
+    const defaultText = styleParams.isStoreGallery ? 'Buy Now' : 'Click here';
```

Comment:
the default is 'Click Here', the art-store wrapper should handle the default there